platform/darwin/src/MGLShape.mm

```diff
@@ -72,6 +73,7 @@ - (void)encodeWithCoder:(NSCoder *)coder
 {
     [coder encodeObject:_title forKey:@"title"];
     [coder encodeObject:_subtitle forKey:@"subtitle"];
+    [coder encodeBool:_enabled forKey:@"enabled"];
```

Comment:
If two MGLShape objects are identical in every way except that one is enabled and one isn’t, the two are considered object-equal according to this PR. Is that correct, or should object equality also consider enabled state? Philosophically, a shape’s coordinates – its form – is intrinsic to the shape. Could the same be said of its interactivity? If so, what about the feature’s cursor (which is currently implemented on macOS as `MGLAnnotationImage.cursor`)?